stock_move_not_merge_by_dest_moves/models/stock_move.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2025 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+
+from odoo import api, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    @api.model
+    def _prepare_merge_moves_distinct_fields(self):
+        """Do not merge moves that goes to different destination moves"""
+        fields = super()._prepare_merge_moves_distinct_fields()
+        fields.append("move_dest_ids")
```

Comment:
I'm wondering if this is something we'd like be able to be optionable in the operation type

Reply:
Well, I'm thinking in the rest of picking workflows were maybe this isn't necessary (e.g.: receptions)